dotnet/Directory.Packages.props

```diff
@@ -21,8 +21,6 @@
     <PackageVersion Include="Microsoft.Azure.Kusto.Data" Version="11.3.5" />
     <PackageVersion Include="Microsoft.Azure.WebJobs.Extensions.OpenApi" Version="1.5.1" />
     <PackageVersion Include="Microsoft.Bcl.HashCode" Version="1.1.1" />
-    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
-    <PackageVersion Include="Microsoft.Bcl.Numerics" Version="8.0.0" />
```

Comment:
Looks like these BCL changes have introduced some build issues.  See https://github.com/microsoft/semantic-kernel/actions/runs/9595519096/job/26549635542?pr=6479

```
/usr/share/dotnet/sdk/8.0.302/Microsoft.Common.CurrentVersion.targets(2401,5): error MSB3277: Found conflicts between different versions of "Microsoft.Bcl.AsyncInterfaces" that could not be resolved. [/home/runner/work/semantic-kernel/semantic-kernel/dotnet/src/IntegrationTests/IntegrationTests.csproj]
/usr/share/dotnet/sdk/8.0.302/Microsoft.Common.CurrentVersion.targets(2401,5): error MSB3277: There was a conflict between "Microsoft.Bcl.AsyncInterfaces, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" and "Microsoft.Bcl.AsyncInterfaces, Version=8.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51". [/home/runner/work/semantic-kernel/semantic-kernel/dotnet/src/IntegrationTests/IntegrationTests.csproj]
/usr/share/dotnet/sdk/8.0.302/Microsoft.Common.CurrentVersion.targets(2401,5): error MSB3277:     "Microsoft.Bcl.AsyncInterfaces, Version=6.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" was chosen because it was primary and "Microsoft.Bcl.AsyncInterfaces, Version=8.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51" was not. [/home/runner/work/semantic-kernel/semantic-kernel/dotnet/src/IntegrationTests/IntegrationTests.csproj]
```

Reply:
@westey-m should be addressed with latest commit.